dev/com.ibm.ws.microprofile.metrics.private/bnd.bnd

```diff
@@ -23,6 +23,7 @@ IBM-Authorization-Roles: com.ibm.ws.management
 IBM-Web-Extension-Processing-Disabled: true
 
 Web-ContextPath: @privateMetricsURL
+OL-Virtual-Host: ${admin.virtual.host}
```

Comment:
What does the server.xml config look like to set this?  Are we intending for this to be configurable by the user?  Or are we going to have some default value set?  If we have some default value set for admin.virtual.host where is that set?  Or is the default `null` and when null then we just fallback to the default virtual host?